.github/workflows/benchmark-paradedb.yml

```diff
@@ -1,47 +1,47 @@
-# workflows/benchmark-paradedb.yml
```

Comment:
why remove this?

Reply:
Reverted now; I thought I pushed this commit only on my forked repo, but it also added it to this PR, sorry. 
Did it to stop getting daily notifications that the run failed 😅